orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/ClientRegistry.java

```diff
@@ -232,6 +233,20 @@ public ClientRegistry withSubjectType(String subjectType) {
         return this;
     }
 
+    @DynamoDbAttribute("IsActive")
+    public boolean getIsActive() {
+        return isActive;
```

Comment:
Conventionally, methods that return booleans are named `isXXX` rather than `getIsXXX` because it reads more fluently by consumers.

Is there a reason why we're using this naming approach?

Reply:
No reason, just new to OOP, thanks for the tip:)